### DISCUSSION TITLE
SpiralLinux 12.231120 release

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#364] [Q&A] [answered]
`All editions: Now using systemd-timesyncd instead of chrony as recommended by Debian.`

How to get it if using a earlier version?

**Comment by geckolinux** [comment, marked as answer]
Hi there, you can uninstall `chrony` and install `systemd-timesyncd` .

**Reply by AlphaElwedritsch** [reply]
it can also just be simple ;)

**Comment by AlphaElwedritsch** [comment]
Did it
Easy
Purged chrony and installed timesyncd
Configured my time server. That's it...
